attestation-service/verifier/src/az_snp_vtpm/mod.rs

```diff
@@ -43,6 +44,24 @@ impl AzSnpVtpm {
     }
 }
 
+pub(crate) fn extend_claim_with_tpm_quote(
+    claim: &mut TeeEvidenceParsedClaim,
+    quote: &Quote,
+) -> Result<()> {
+    let Value::Object(ref mut map) = claim else {
+        bail!("failed to extend the claim, not an object");
+    };
+
+    let mut tpm_values = serde_json::Map::new();
+    for (i, pcr) in quote.pcrs_sha256().enumerate() {
+        tpm_values.insert(format!("pcr{:02}", i), Value::String(hex::encode(pcr)));
+    }
+    debug!("extending claim with TPM quote: {:#?}", tpm_values);
+    map.insert("tpm".to_string(), Value::Object(tpm_values));
```

Comment:
We'd better add a document about the parsed claims for az-x-tpm platforms like [SGX](https://github.com/confidential-containers/kbs/blob/main/attestation-service/docs/parsed_claims.md#intel-sgx).

Reply:
makes sense, will add documentation.